config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get '/dishes/:id', to: 'dishes#show'
+  patch '/dishes/:id', to: 'dish_ingredients#new'
```

Comment:
Should be: `post '/dishes/:id/ingredients', to: 'dish_ingredients#create'`

A `patch` verb with a `new` action is not RESTful.